token/program-2022/src/instruction.rs

```diff
@@ -576,7 +580,13 @@ impl TokenInstruction {
                     decimals,
                 }
             }
-            21 => Self::GetAccountDataSize,
+            21 => {
+                let mut extension_types = vec![];
+                for chunk in rest.chunks(size_of::<ExtensionType>()) {
```

Comment:
@CriesofCarrots - I just wanted to point out that I'm not serializing the vector length here.  On deserialization, the code just pulls off `u16`'s until there are no more. I think is is ok though

Reply:
Yeah, I noticed that. I think it is okay also. Thanks for highlighting it. Hmm, should we add a comment to make it more explicit?

Reply:
I feel it's prolly ok as is, until somebody trips over it :)